README.md

```diff
@@ -1,8 +1,8 @@
 # Majority Judgment Library for Java
 
 [![MIT](https://img.shields.io/github/license/MieuxVoter/majority-judgment-library-java)](./LICENSE.md)
-![Build Status](https://img.shields.io/github/workflow/status/MieuxVoter/majority-judgment-library-java/Java%20CI%20with%20Maven)
-![Release](https://img.shields.io/github/v/release/MieuxVoter/majority-judgment-library-java?sort=semver)
+[![Build Status](https://img.shields.io/github/workflow/status/MieuxVoter/majority-judgment-library-java/Java%20CI%20with%20Maven)](https://github.com/MieuxVoter/majority-judgment-library-java/actions)
+[![Release](https://img.shields.io/github/v/release/MieuxVoter/majority-judgment-library-java?sort=semver)](https://github.com/MieuxVoter/majority-judgment-library-java/releases)
 [![Join the Discord chat at https://discord.gg/rAAQG9S](https://img.shields.io/discord/705322981102190593.svg)](https://discord.gg/rAAQG9S)
 
 Test-driven java library to help deliberate using Majority Judgment.
```

Comment:
can you link to MJ to the website? like this people asking for more context would be served :)

Reply:
I have, but I really dislike how the MV website builds links : https://mieuxvoter.fr/index.php/decouvrir/?lang=en   It's not future-proof.  We've got to remake that website as well, it's getting old and decrepit !

Reply:
Yup... We'll definitively have to this part as well! But I think we'll first make progress on the app + external web sites (like jm2022.fr)